apps/clippo/clippo.yml

```diff
@@ -0,0 +1,18 @@
+name: kalk
```

Comment:
This should be `Clippo`, right?

Reply:
Hi zeke, 

not really, I like it lower case, but if necessary I can live with the upper case as well.

Reply:
Oops. sorry. Misunderstood your question. Yes, in the clippo.yaml it should be clippo instead of kalk.
I will fix that.

Reply:
Ok, should be fixed now. 
Thanks for the hint.